styles/profile.css

```diff
@@ -0,0 +1,83 @@
+@import "_base.css";
+@import "colors.css";
+
+.outerGrid {
+  display: grid;
+  grid-template-rows: 60px auto 75px;
+  height: 100vh;
+}
+
+.header {
+  padding: 10px;
+  text-align: center;
+  font-size: large;
+  background-image: linear-gradient(to right, #ee0979, #ff6a00);
+}
+
+.content {
+  padding: 10px;
+  background: var(--primary-bg-color);
+  text-align: center;
+}
+
+.profile__Picture {
+  border-radius: 30px;
+  height: 20%;
```

Comment:
I'd be careful with using % in height. That can lead to strange result. 

Look at the aspect-ratio property that should get you what you want to achieve: 
https://developer.mozilla.org/en-US/docs/Web/CSS/aspect-ratio